homeassistant/components/comelit/config_flow.py

```diff
@@ -159,6 +159,43 @@ async def async_step_reauth_confirm(
             errors=errors,
         )
 
+    async def async_step_reconfigure(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle reconfiguration of the device."""
+        reconfigure_entry = self._get_reconfigure_entry()
+        if not user_input:
+            return self.async_show_form(
+                step_id="reconfigure", data_schema=user_form_schema(user_input)
+            )
```

Comment:
I would recommend to invert this and just re-use the `self.async_show_form` at line 193

Reply:
I used the same flow as other integrations.

The issue here is that different reviewers has different approaches ;-)
So getting all code on the same page is really hard :-(

Reply:
It's also a recommendation with the intention to deduplicate code :)